pkg/component/controller/etcd_member_reconciler.go

```diff
@@ -153,8 +154,8 @@ func (e *EtcdMemberReconciler) waitForCRD(ctx context.Context) error {
 				return retryAfter, nil
 			}
 
-			retryAfter := 10 * time.Second
-			log.WithError(err).Errorf(
+			retryAfter = 10 * time.Second
+			log.WithError(e).Errorf(
```

Comment:
Removing the named `err` return parameter made the name `err` available again, When renaming the `cbErr` func parameter to `err`, things fall into place a bit nicer, and the whole change would boil down to a one-liner. When not renaming `cbErr` to `err`, I'd suggest to at least rename `e` to `err`, though.